packages/farmify-server/controllers/users.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-shadow */
+const { validationResult } = require('express-validator');
+const debug = require('debug')('farmify-server:server');
+const db = require('../db');
+const { hash } = require('../lib/helpers');
+
+
+module.exports = {
+  register: (req, res, next) =>
+  {
+    try
+    {
+      const {
+        firstName, lastName, email, phone, password, address, city, state, country, userType,
+      } = req.body;
+      const errors = validationResult(req);
+      if (!errors.isEmpty())
+      {
+        res.status(422).send({ errors: errors.array() });
+      }
+      else
+      {
+        const hashedPassword = hash(password);
+        if (hashedPassword)
+        {
+          db.query(`INSERT INTO users (firstName, lastName, email, phone, password, address, city, state, country, userType)
```

Comment:
Good job on this PR. You may want to refactor these DB specific statements into a separate function going forward.